SpatialGDK/Source/SpatialGDK/Private/Interop/GlobalStateManager.cpp

```diff
@@ -119,10 +119,10 @@ void UGlobalStateManager::TrySendWorkerReadyToBeginPlay()
 	// AddComponent. This is important for handling startup Actors correctly in a zoned
 	// environment.
 	const bool bHasReceivedStartupActorData =
-		StaticComponentView->HasComponent(GlobalStateManagerEntityId, SpatialConstants::STARTUP_ACTOR_MANAGER_COMPONENT_ID);
+		ViewCoordinator->HasComponent(GlobalStateManagerEntityId, SpatialConstants::STARTUP_ACTOR_MANAGER_COMPONENT_ID);
```

Comment:
why doesn't this ask `WellKnownEntitySystem` (who I believe handles all GSM entity related adventures)